packages/i18n/src/locales/fr/user.json

```diff
@@ -156,5 +156,15 @@
       "passwordValidation": "Password must contain at least 8 characters, including a number, a lowercase and an uppercase (fr)",
       "mustMatch": "Passwords must match (fr)"
     }
+  },
+  "emailVerification": {
+    "messages": {
+      "alreadyVerified": "Email is already verified. Redirecting to home page in 5 seconds (fr)",
```

Comment:
we should be displaying the countdown as 5, 4, 3, 2, 1 seconds in the ui so we cannot define 5 seconds here. We can keep this and do the countdown in a separate PR.